tests/tt_eager/python_api_testing/non_working_unit_tests/wormhole/test_dispatch_mode_transpose_nh.py

```diff
@@ -20,7 +21,7 @@
 
 def tt_transpose_nh(x, device, dtype, layout, input_mem_config, output_mem_config):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttl.tensor.transpose(t0, 0, 2, output_mem_config=output_mem_config)
+    t1 = ttl.tensor.transpose(t0, 0, 2, memory_config=output_mem_config)
```

Comment:
Did you mean `ttnn.transpose` here?

Reply:
oops ya I changed it , thanks @yieldthought